packages/gatsby-theme-vtex/src/components/Minicart/Drawer/Content.tsx

```diff
@@ -1,3 +1,63 @@
-import { MinicartContent } from '@vtex/store-ui'
+import React, { FC } from 'react'
+import { Box, Flex, Text } from 'theme-ui'
 
-export const HeaderMinicartDrawerContent = MinicartContent
+import MinicartDelete from './Delete'
+import MinicartQuantity from './Quantity'
+
+export interface DataItem {
+  id: string | number
```

Comment:
Why `string | number`? doesn't make it harder to perform operations on top of this data?

Reply:
yes, I fixed it.